Interop.md

```diff
@@ -23,3 +23,8 @@ Creating snapshots of a VM which has containers running is not supported.
 - If a container is running and a snapshot is taken then if the container exits, the volume is not detached. VC shows an error that the volume is part of a snapshot and hence can't be removed from the VM config. For this reason, snapshotting a VM after starting one or more containers isn't supported as it leaves the volume attached to the VM. [#1649](https://github.com/vmware/docker-volume-vsphere/issues/1649)
 
 
+**Site Recovery Manager[SRM]**
+
+User created vmgroup is not supported with SRM.
+
+- After disaster recovery finishes, ```esxcli volume ls``` shows 'N/A' (volume<->vmgroup relation) for the user created vmgroup's volumes at the secondary site and all VMs that were part of user created vmgroup will fall into `_DEFAULT` vmgroup and will have access to volumes of `_DEFAULT` vmgroup - in short it looses the volumes that were part of user created vmgroup and gains access to volumes of `_DEFAULT` vmgroup. [#1786](https://github.com/vmware/docker-volume-vsphere/issues/1786)
```

Comment:
I think the root cause is not this. The volume was created from a vm belongs to user created vmgroup. After SRM, the vmgroup configuration is not replicated to the destination, and therefore, it cannot find the which vmgroup those volumes belong to. It has nothing related with _DEFAULT vmgroup.